packages/web3/src/entitlement.ts

```diff
@@ -677,7 +680,7 @@ export function createOperationsTree(
             if (i + 1 < operations.length) {
                 newOperations.push({
                     opType: OperationType.LOGICAL,
-                    logicalType: LogicalOperationType.AND,
+                    logicalType: logicalOp as LogicalOperationType.AND | LogicalOperationType.OR,
```

Comment:
is this right? I was expecting just `logicalType: logicalOp` here...

Reply:
linter was complaining that you cannot assign type `LogicalOperation` to `LogicalOperation.AND | LogicalOperation.OR` - I think this is because `Operation` is an OR of `AndOperation and OrOperation` (and some other types), and each of them have an explicit value for their logical operation type?

Reply:
This is the built-in linter with vs code, I can try with cli tool to confirm

Reply:
`yarn build` fails locally if I remove this.